crypto-core/src/main/java/com/palantir/crypto2/cipher/SeekableCipher.java

```diff
@@ -18,33 +18,47 @@
 
 import com.palantir.crypto2.keys.KeyMaterial;
 import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import org.apache.commons.crypto.cipher.CryptoCipher;
 
 /**
- * Provides access to a {@link Cipher} with the ability to {@link #seek} the Cipher.
+ * Provides access to a {@link CryptoCipher} with the ability to {@link #seek} the Cipher.
  * // TODO(jellis): add note about using SeekableCipher streams instead of this directly
  */
 public interface SeekableCipher {
 
     /**
-     * Initialize the underlying {@link Cipher} to either {@link Cipher#ENCRYPT_MODE} or {@link Cipher#DECRYPT_MODE}.
+     * Initialize the underlying {@link CryptoCipher} to either {@link Cipher#ENCRYPT_MODE} or
+     * {@link Cipher#DECRYPT_MODE}.
      */
-    Cipher initCipher(int opmode);
+    CryptoCipher initCipher(int opmode);
```

Comment:
Is that because it's changing the return value of a method in a public interface?